src/Uno.UWP/UI/ViewManagement/ApplicationView.iOS.cs

```diff
@@ -8,41 +8,46 @@
 
 namespace Windows.UI.ViewManagement
 {
-    partial class ApplicationView
+	partial class ApplicationView
 	{
 		private static bool UseSafeAreaInsets => UIDevice.CurrentDevice.CheckSystemVersion(11, 0);
 
 		internal void SetCoreBounds(UIKit.UIWindow keyWindow, Foundation.Rect windowBounds)
 		{
-            var statusBarHeight = UIApplication.SharedApplication.StatusBarFrame.Size.Height;
+			var inset = UseSafeAreaInsets
+					? keyWindow.SafeAreaInsets
+					: new UIEdgeInsets(0, 0, 0, 0);
 
-			UIEdgeInsets inset = new UIEdgeInsets(statusBarHeight, 0, 0, 0);
 			// Not respecting its own documentation. https://developer.apple.com/documentation/uikit/uiview/2891103-safeareainsets?language=objc
-			// iOS returns all zeros for SafeAreaInsets on non-iPhoneX phones. (ignoring nav bars or status bars)
-			// For that reason, we will set the window's visible bounds to the SafeAreaInsets only for iPhones with notches,
-			// other phones will have insets that consider the status bar
-			if (UseSafeAreaInsets)
+			// iOS returns all zeros for SafeAreaInsets on non-iPhones. (ignoring nav bars or status bars)
```

Comment:
Was the previous comment wrong? Does SafeAreaInsets include the status bar height for iPhones other than X? Otherwise it seems like we're leaving it out for anything other than iPads and notched iPhones.

Reply:
Actually it does. SafeAreaInsets include the status bar on all phones. For some reason, it is not working for tablets. I just have no clue why. But yeah, all iPhones have it (tested on devices and simulators).

Reply:
And just to confirm, that's the case on all versions of iOS (or at least 10+)?

Reply:
Yes :)